keyboards/splitography/keymaps/beaklmu/common/init.h

```diff
@@ -0,0 +1,52 @@
+
+// ....................................................................... Audio
+
+void matrix_init_user(void)
+{
+#ifdef STENO_ENABLE
+  steno_set_mode(STENO_MODE_BOLT);          // or STENO_MODE_GEMINI
+#endif
+#ifdef AUDIO_ENABLE
+  startup_user();
+#endif
```

Comment:
This is handled by `quantum.c` (core), and isn't needed.  In fact, this overrides default behavior.